imap_processing/ultra/l0/ultra_utils.py

```diff
@@ -316,7 +317,11 @@ def append_ccsds_fields(decom_data: dict, ccsds_data_object: object):
     ccsds_data_object : object
         CCSDS data object.
     """
-    for field in fields(ccsds_data_object.__class__):
+    for field in fields(ccsds_data_object.__class__):  # type: ignore[arg-type]
```

Comment:
Here is a relevant stackoverflow I found about this: https://stackoverflow.com/questions/77071473/where-can-i-import-dataclassinstance-for-mypy-check

I couldn't seem to get this to work. Feel free to try yourself, but if its too much of a hassle I think we can keep the `ignore`

Reply:
I think the return error is because of the changed to how process_function was implemented. I can simply change what the function returns to reflect what the error returns.

Reply:
@bourque 
I was able to successfully import DataclassInstance and resolve the mypy errors occurring. However, I am unsure if the type changes I made will negatively affect the code and would like them double checked.